components/post_view/message_attachments/message_attachment/index.ts

```diff
@@ -19,12 +22,13 @@ function mapStateToProps(state: GlobalState) {
 
 type Actions = {
     doPostActionWithCookie: (postId: string, actionId: string, actionCookie: string, selectedOption?: string | undefined) => Promise<ActionResult>;
+    openModal: <P>(modalData: ModalData<P>) => void;
 }
 
 function mapDispatchToProps(dispatch: Dispatch<GenericAction>) {
     return {
-        actions: bindActionCreators<ActionCreatorsMapObject<ActionFunc>, Actions>({
-            doPostActionWithCookie,
+        actions: bindActionCreators<ActionCreatorsMapObject<ActionFunc | GenericAction>, Actions>({
```

Comment:
Please let me know if it is OK to add `GenericAction`

Reply:
Yep, that would work. You could also use the `Action` type from that same file instead of `ActionFunc | GenericAction`, but we currently use both ways of doing that in various places throughout the code base, so either is fine.